configure.ac

```diff
@@ -109,6 +109,17 @@ if test "$ZLIB" = "no"; then
     exit 1
 fi
 
+# liblzma
+AC_CHECK_HEADER(lzma.h,AC_DEFINE([HAVE_LIBLZMA],[1],[liblzma available]),[AC_ERROR(lzma.h not found ...)])
```

Comment:
Using AC_ERROR seems wrong here:
```
checking for lzma.h... no
configure: error: lzma.h not found ...
configure: error: ./configure failed for libhtp
ERROR: Job failed: exit code 1
```

Reply:
I'm quickly dumped in the checks from suricata, but then I see the next issue: make check fails, presumably due to lzma dependencies there?

Reply:
Thank you again Victor for the review

Reply:
Side note, I think the lzma checks should be fixed now in master as part of
https://github.com/OISF/suricata/commit/b98c28a60de49d71631ef81a273b88e1c7d3a028#diff-67e997bcfdac55191033d57a16d1408a